### PR TITLE
Workflow Automated Test - LRQA-67563 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase [hunk 1]
```diff
@@ -77,15 +77,34 @@ definition {
 	}
 
 	@description = "LPS-86642 - Verify that when there is no SLA defined for the process, the card should display only the Total Pending column populated"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test OnlyTotalPendingColumnIsPopulatedWhenNoSLA {
 		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase:82]: Hello @AfonsoHSantos, Can you remove this `WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");`, please?

Thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase (hunk 1 above)
Comment (AfonsoHSantos) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase]: fixed.

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
[Sent to BChan](https://github.com/CalumR23/liferay-portal/pull/19)

**Pull request closed without merging**